pc2ccs/pc2v9
# I170: now loads reject.ini from CDP/config dir - automation

### Description of what the PR does

The PR fix

### Issue which the PR fixes

Issue 170

### Environment in which the PR was developed (OS,IDE, Java version, etc.)

Java version 1.8.0_121
OS: Windows 10 10.0 (amd64)

### Precise steps for _testing_ the PR (i.e., how to demonstrate that it works correctly)

Here is the load reject.ini precedence
1 - Load judgements from CDP/config/reject.ini
2 - Load default judgements
3 - Load judgements from ./reject.ini in directory where server started

All three combinations have been tested by the developer, multiple times.

Tests done in reverse order in order to test precedence.

------------
Test 3 - Load default judgements

1. run bin/pc2reset 
     ```
     bin/pc2reset
     ```
1. Start server, start admin with no ./reject.ini  or config/reject.ini
     ```
     bin\pc2server --login s --contestpassword contest
     bin\pc2admin --login r
     ```
1. In admin select Judgements tab

Expected Default judgements

Yes acronym=AC 
No - Compilation Error acronym=CE 
No - Run-time Error acronym=RTE 
No - Time Limit Exceeded acronym=TLE 
No - Wrong Answer acronym=WA 
No - Excessive Output acronym=EO 
No - Output Format Error acronym=OFE 
No - Other - Contact Staff acronym=OCS 

------------
Test 2 - Load judgements from reject.ini in directory where server started

1. copy samps reject.ini
     ```
     copy samps\reject.ini .
     ```
1. run bin/pc2reset 
     ```
     bin/pc2reset
     ```
1. Start server, start admin with no ./reject.ini  or config/reject.ini
     ```
     bin\pc2server --login s --contestpassword contest
     bin\pc2admin --login r
     ```
1. In admin select Judgements tab

Expected
(Note new/added: 'No - Reread Problem Statement' acronym=WA007)

'Yes' acronym=AC
'No - Compilation Error' acronym=WA001
'No - Run-time Error' acronym=WA002
'No - Time Limit Exceeded' acronym=WA003
'No - Wrong Answer' acronym=WA004
'No - Excessive Output' acronym=WA005
'No - Output Format Error' acronym=WA006
'No - Reread Problem Statement' acronym=WA007
'No - Other - Contact Staff' acronym=WA008

------------
Test 1 - Load judgements from CDP/config/reject.ini

Keeping ./reject.ini will also test the precedence of using config/reject.ini first.

Load from sample CDP problemflagtest

1. run bin/pc2reset 
     ```
     bin/pc2reset
     ```
1. Start server, start admin with ./reject.ini 
     ```
     bin\pc2server --login s --contestpassword contest --load problemflagtest
     bin\pc2admin --login r
     ```
1. In admin select Judgements tab

Expected
(Note: 'No - Various Differences' acronym=VD  )

  'Yes/Accepted' acronym=AC 
  'No - Compilation Error' acronym=CE 
  'No - Run-time Error' acronym=RTE 
  'No - Time Limit Exceeded' acronym=TLE 
  'No - Wrong Answer' acronym=WA 
  'No - Excessive Output' acronym=EE 
  'No - Output Format Error' acronym=EFE 
  'No - Various Differences' acronym=VD 
  'No - Other - Contact Staff' acronym=CS
